### ISSUE TITLE
Invalid "Missing key prop for element in array" error

### ISSUE THREAD

**Issue by Jnp95** [yannickcr/eslint-plugin-react#3229]
Hello,

Since serveral days, the error "Missing key prop for element in array" appears everytime on every children tag, that is invalid:
```jsx
export default function ESlintTest() {
  return (
    <div>
      <div>Test</div>
      {/* Error here above */}
    </div>
  );
}
```

![image](https://user-images.githubusercontent.com/73538541/156202271-71daa630-1399-4741-af8f-b2b671cac565.png)

**Comment by ljharb** [comment]
Perhaps you’re still on v7.29.0? This should be fixed in v7.29.2.

**Comment by Jnp95** [comment]
That's ok ! We used the old version 7.29.0, the version 7.29.2 fixed the issue!
Thank you!
